package.json

```diff
@@ -56,7 +56,8 @@
     "coverage": "gulp coveralls",
     "precommit-msg": "echo 'Pre-commit checks...' && exit 0",
     "precommit-test": "REPORTER=dot gulp once",
-    "lint": "gulp eslint"
+    "lint": "gulp eslint",
+    "postinstall": "git submodule update --init && rm -rf ~/Library/Developer/Xcode/DerivedData/WebDriverAgent-*"
```

Comment:
probably only want to run this in the context of a git checkout? don't want it to be done as part of an `npm install appium-xcuitest-driver` or `npm install appium` right?